share/dkg/pedersen/dkg.go

```diff
@@ -18,8 +18,12 @@ type Suite vss.Suite
 type DistKeyShare struct {
 	// Coefficients of the public polynomial holding the public key
 	Commits []kyber.Point
-	// Share of the distributed secret
+	// Share of the distributed secret which is a private information
```

Comment:
Remove the "a": which is private information

End complete sentences with a period. (Also for Commits.)